package.json

```diff
@@ -12,7 +12,7 @@
   },
   "scripts": {
     "test": "mocha",
-    "build": "rollup -c -f cjs -o dist/pluginutils.cjs.js && rollup -c -f es6 -o dist/pluginutils.es6.js",
+    "build": "rollup -c -f cjs -o dist/pluginutils.cjs.js && rollup -c -f es -o dist/pluginutils.es.js",
```

Comment:
Can you provide some source lines @TrySound? Do have empathy that you know this like the back of your hand. I'm still rounding up all the pieces. 

Thanks again in advance!

![](https://media.giphy.com/media/j1QQj6To9Pbxu/giphy.gif)